06.bowling_object/game.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require './shot'
+require './frame'
+
+class Game
+  def initialize(score)
+    @frames = parse_frames(score)
+  end
+
+  def calc
+    total_score = 0
+    @frames.each_with_index do |frame, frame_count|
```

Comment:
```suggestion
    @frames.each_with_index.sum do |frame, frame_count|
```

のようにすると `total_score`をなくせます

Reply:
each.sumに修正しました